--- /usr/portage/www-servers/nginx/nginx-0.7.62.ebuild  2009-09-18 19:36:47.000000000 +0000
+++ /usr/local/portage/www-servers/nginx/nginx-0.7.62.ebuild    2009-10-02 10:47:20.000000000 +0000
@@ -11,13 +11,14 @@
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS="amd64 ~ppc x86 ~x86-fbsd"
-IUSE="addition debug fastcgi flv imap pcre perl random-index ssl status sub webdav zlib"
+IUSE="addition debug fastcgi flv imap passenger pcre perl random-index ssl status sub webdav zlib"

 DEPEND="dev-lang/perl
        pcre? ( >=dev-libs/libpcre-4.2 )
        ssl? ( dev-libs/openssl )
        zlib? ( sys-libs/zlib )
-       perl? ( >=dev-lang/perl-5.8 )"
+       perl? ( >=dev-lang/perl-5.8 )
+       passenger? ( www-misc/passenger )"

 pkg_setup() {
        ebegin "Creating nginx user and group"
@@ -60,6 +61,8 @@
        use webdav      && myconf="${myconf} --with-http_dav_module"
        use sub         && myconf="${myconf} --with-http_sub_module"
        use random-index        && myconf="${myconf} --with-http_random_index_module"
+       use passenger   && myconf="${myconf} --add-module=`passenger-config --root`/ext/nginx"
+

        tc-export CC
        ./configure \
